app/components/contact_details_component.rb

```diff
@@ -0,0 +1,21 @@
+class ContactDetailsComponent < ActionView::Component::Base
+  def initialize(contact_details_form:)
+    @contact_details_form = contact_details_form
+    @contact_details_form_rows = convert_to_rows
+  end
+
+private
+
+  def convert_to_rows
+    [phone_number_row]
+  end
+
+  def phone_number_row
+    {
+      key: I18n.t('application_form.contact_details.phone_number.label'),
+      value: @contact_details_form.phone_number,
+      action: t('application_form.contact_details.phone_number.change_action'),
+      change_path: Rails.application.routes.url_helpers.candidate_interface_contact_details_edit_path,
```

Comment:
```suggestion
      change_path: candidate_interface_contact_details_edit_path,
```

Not necessary in `ActionView::Component` :)

Reply:
When this is removed, Rails throws an error. We're going leave it as is for now.